clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinatorTest.java

```diff
@@ -283,7 +283,7 @@ public void testGroupMaxSizeExceptionIsFatal() {
 
         mockClient.prepareResponse(joinGroupFollowerResponse(defaultGeneration, memberId, JoinGroupResponse.UNKNOWN_MEMBER_ID, Errors.GROUP_MAX_SIZE_REACHED));
 
-        RequestFuture<ByteBuffer> future = coordinator.sendJoinGroupRequest();
+        RequestFuture<ByteBuffer> future = coordinator.sendJoinGroupRequest(coordinator.generation);
```

Comment:
Ditto here, we can still use the internal generation id no matter if there's a race condition changing it.

Reply:
Fixed